package.json

```diff
@@ -36,6 +36,7 @@
     "build": "npm run types:network && npm run js:network && npm run images:network",
     "watch": "npm run images:network && rollup --watch --config rollup.config.js",
     "watch-dev": "npm run watch-dev",
+    "generate-examples-index": "node -r ./babel.mocha.js generate-example-index.ts -c mynetwork -w https://visjs.github.io/vis-network -lis",
```

Comment:
We should think about calling this from within the CI. This could be done together with #95

Reply:
Yeah, when #95 is merged I'll add this to it.

Reply:
Or the other way around if this is merged first.